packages/apps/docs/src/components/Markdown/Link/Link.tsx

```diff
@@ -0,0 +1,23 @@
+import { default as NextLink } from 'next/link';
+import React, { FC, ReactNode } from 'react';
+
+interface IProp {
+  children: ReactNode;
+  href: string;
+}
+
+export const Link: FC<IProp> = ({ children, href, ...props }) => {
+  if (typeof children === 'string' && children.includes('http')) {
```

Comment:
did you test this?
- you need to check the href for http
- it should have a `!`

Reply:
you're right! thing is, even though is discouraged, next.js `<Link>` component works with external links just fine, so testing was showing alright for it